src/device-manager/cocoa/NLWdmClient.h

```diff
@@ -59,6 +59,14 @@ typedef void (^WdmClientFailureBlock)(id owner, NSError * error);
  */
 - (void)setNodeId:(uint64_t)nodeId;
 
+/**
+ * setEventFetchingTimeout when refreshDataWithEvents
+ *
+ * @param timeout in seconds
+ *
+ */
+- (void)setEventFetchingTimeout:(uint32_t)timeout;
```

Comment:
Can the timeout just be added as a parameter to refreshDataWithEvents()? It's less state to keep around, and makes it implicitly clear what operation it belongs to.

Reply:
I have just pushed another commit (1a901cc) for this comment, PTAL

Reply:
Thanks, do we still need the separate setEventFetchingTimeout() in this case?